src/tike/ptycho/probe.py

```diff
@@ -235,6 +250,8 @@ def resample(self, factor: float, interp) -> ProbeOptions:
             update_period=self.update_period,
             init_rescale_from_measurements=self.init_rescale_from_measurements,
             probe_photons=self.probe_photons,
+            probe_wavelength=self.probe_wavelength,
+            probe_FOV_lengths=self.probe_FOV_lengths,
             force_orthogonality=self.force_orthogonality,
```

Comment:
Use caution when handling unit sizes in the resample functions. I think this might not be correct because when you resample the grid, the real unit sizes of one pixel will change.